projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -364,12 +364,12 @@ function ( $parent_file ) use ( $menu_slug ) {
 	/**
 	 * Adds Appearance menu.
 	 *
-	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
+	 * @param Bool   $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
+	 * @param String $customize_slug Whether customizer links should point to Calypso or wp-admin.
 	 */
-	public function add_appearance_menu( $wp_admin = false ) {
+	public function add_appearance_menu( $wp_admin = false, $customize_slug ) {
```

Comment:
> Declaring a required parameter after an optional one is deprecated since PHP 8.0. Parameter $wp_admin is optional, while parameter $customize_slug is required. (PHPCompatibility.FunctionDeclarations.RemovedOptionalBeforeRequiredParam.Deprecated)

I would prefer to leave this as is cause:
- refactoring all the calls will introduce some noise which would not let easy cherry picking
- this code WILL be refactored in follow up PR https://github.com/Automattic/jetpack/pull/18586

Reply:
It seems though, that tests do not pass. Let me think about it

Reply:
Fixed d570570